.travis.yml

```diff
@@ -27,10 +27,10 @@ jobs:
       after_success:
     - stage: test
       env:
-        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="event_sale_reservation" MAKEPOT="1"
```

Comment:
Why are you excluding the module from tests?

Reply:
I have changed the file again to add the addon to the tests.
This addon is incompatible with: `event_registration_multi_qty` + `event_sale_registration_multi_qty` and for that reason it is excluded from Travis, and then only this addon is tested.

Related to: https://github.com/OCA/event/commit/10ea6248ad30e1286dde06a887d53da57b45a539

Reply:
Well, you have to make them compatible. Start putting the problem, and seeing how to compatibilize them. Usually, this is done nullifying the modified behavior if in test mode and not detecting a specific context that is passed in the tests.

Reply:
The comment that appears to exclude the addons is: 

> These modules makes totals wrong because they depend currently on count and not sum of qtys; integrating with them would require a glue module

It seems that it is not only related to the tests, I try to solve the compatibilities anyway?

Reply:
Give it a try to see why the comment.